docs/concepts/Smapi-Command.md

```diff
@@ -166,6 +300,282 @@ Gets detailed information about an upload which was created for a specific catal
+    <dt>--debug</dt>
+    <dd markdown="span">When you include this option, ASK CLI shows debug messages in the output of the command.</dd>
+</dl>
+
+### set-subscription-for-development-events
+
+Updates the mutable properties of a subscription. This needs to be authorized by the client&#x2F;vendor who created the subscriber and the vendor who publishes the event. The subscriberId cannot be updated.
+
+`set-subscription-for-development-events` command format:
+
+`$ askx smapi set-subscription-for-development-events <--subscription-id <subscription-id>> [--name <name>] [--events <events>] [-p| --profile <profile>] [--debug]`
+
+**Options**
+
+<dl>
+    <dt>--subscription-id <subscription-id></dt>
+    <dd markdown="span">[REQUIRED] Unique identifier of the subscription.</dd>
+    <dt>--name <name></dt>
+    <dd markdown="span">[OPTIONAL] Name of the subscription.</dd>
+    <dt>--events <events></dt>
+    <dd markdown="span">[OPTIONAL] The list of events that the subscriber should be notified for.</dd>
+    <dt>-p, --profile <profile></dt>
+    <dd markdown="span">The ASK CLI profile to use. When you don't include this option, ASK CLI uses the default profile.</dd>
+    <dt>--debug</dt>
+    <dd markdown="span">When you include this option, ASK CLI shows debug messages in the output of the command.</dd>
+</dl>
+
+### delete-subscription-for-development-events
+
+Deletes a particular subscription. Both, the vendor who created the subscriber and the vendor who publishes the event can delete this resource with appropriate authorization.
+
+`delete-subscription-for-development-events` command format:
+
+`$ askx smapi delete-subscription-for-development-events <--subscription-id <subscription-id>> [-p| --profile <profile>] [--debug]`
+
+**Options**
+
+<dl>
+    <dt>--subscription-id <subscription-id></dt>
+    <dd markdown="span">[REQUIRED] Unique identifier of the subscription.</dd>
+    <dt>-p, --profile <profile></dt>
+    <dd markdown="span">The ASK CLI profile to use. When you don't include this option, ASK CLI uses the default profile.</dd>
+    <dt>--debug</dt>
+    <dd markdown="span">When you include this option, ASK CLI shows debug messages in the output of the command.</dd>
+</dl>
+
+### generate-catalog-upload-url
+
+Generate preSigned urls to upload data.
+
+`generate-catalog-upload-url` command format:
+
+`$ askx smapi generate-catalog-upload-url <-c|--catalog-id <catalog-id>> <--number-of-upload-parts <number-of-upload-parts>> [-p| --profile <profile>] [--debug]`
+
+**Options**
+
+<dl>
+    <dt>-c,--catalog-id <catalog-id></dt>
+    <dd markdown="span">[REQUIRED] Unique identifier of the catalog.</dd>
+    <dt>--number-of-upload-parts <number-of-upload-parts></dt>
+    <dd markdown="span">[REQUIRED] The number of parts the file will be split into. An equal number of presigned upload urls will be generated in response to facilitate each part's upload.</dd>
```

Comment:
nit: `pre-signed`. Change wherever applicable.

Reply:
this is defined from the smapi-sdk .. CLI cannot change it..